dags/openshift_nightlies/tasks/utils/platform_connector.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease, task_group
         self.env = {
             "REL_PLATFORM": self.release.platform,
             "THANOS_RECEIVER_URL": var_loader.get_secret("thanos_receiver_url"),
-            "LOKI_RECEIVER_URL": var_loader.get_secret("loki_receiver_url")
+            "LOKI_RECEIVER_URL": var_loader.get_secret("loki_receiver_url"),
```

Comment:
Is Loki being used? Not related to this PR but just curious?

Reply:
yes, they are getting installed as part of this connector.